pkgs/tools/security/exploitdb/default.nix

```
@@ -0,0 +1,24 @@
+{stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "exploitdb";
+  version = "2021-05-22";
+  src = fetchFromGitHub {
+    owner = "offensive-security";
+    repo = name;
+    rev = version;
+    sha256 = "sha256-cpzAdRAtF47j92u8GHyu6V1ycqtNld+9saCqoboAu+o=";
+  };
+  installPhase = ''
+    mkdir -p $out/bin
+    cp --recursive ./* $out/bin
+    cp ./.searchsploit_rc $out/bin
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/offensive-security/exploitdb";
+    description = "Archive of public exploits and corresponding vulnerable software, developed for use by penetration testers and vulnerability researchers.";
+    license = lib.licenses.gpl2;
```

Headers contains other licenses like [GPL3+](https://github.com/offensive-security/exploitdb/blob/master/exploits/php/webapps/11263.php) or [MIT](https://github.com/offensive-security/exploitdb/blob/master/shellcodes/linux_x86-64/41503.nasm). I assume that there could me more.

```suggestion
    license = with licenses; [ gpl2Plus gpl3Plus mit ];
```